src/common/opencl.c

```diff
@@ -787,7 +782,8 @@ static int dt_opencl_device_init(dt_opencl_t *cl, const int dev, cl_device_id *d
     fclose(f);
     tend = dt_get_wtime();
     tdiff = tend - tstart;
-    dt_print(DT_DEBUG_OPENCL, "[dt_opencl_device_init] kernel loading time: %2.4lf \n", tdiff);
```

Comment:
There are two "subtle" functional differences, dt_print()
1. outputs the current time since start, this is good in most situations to get an idea when something happend. here it just pollutes and redeuces readability imho
2. makes sure all data are completely written to file/console

Maybe a dt_print variant without the time would be a better option?

Reply:
> Maybe a dt_print variant without the time would be a better option?

Yes, I would think so. It bring this feature (no timestamp) for every one without requiring using a bit mask magic :)

Reply:
Any name suggestion?

Reply:
`dt_print_raw` or `dt_print_nts` (no time-stamp) ?

Reply:
I chose the _nts variant